app/views/virtual_workshop_invitation_mailer/invite_coach.html.haml

```diff
@@ -14,15 +14,19 @@
               %td
                 %h3 Hi #{@member.name},
                 %p.lead
-                  Our next virtual #{@workshop.chapter.name} workshop is coming soon, and we’d like to invite you to join us as a coach
+                  We’re excited to invite you to join our upcoming virtual #{@workshop.chapter.name} workshop as a coach!
 
-                %p You will be helping students remotely as they work their way through one of our tutorials or a personal project they need help with. Not to worry, you’ll only be matched with students who you’ll be able to help based on your experience and expertise.
+                %p As a coach, you’ll support students remotely as they work through one of our #{link_to 'tutorials', 'http://tutorials.codebar.io/'} or seek guidance on a personal project. Don’t worry—you’ll only be paired with students whose needs align with your skills and expertise.
```

Comment:
```suggestion
                %p As a coach, you’ll support students remotely as they work through one of our #{link_to 'tutorials', 'http://tutorials.codebar.io/'} or seek guidance on a personal project. Don’t worry — you’ll only be paired with students whose needs align with your skills and expertise.
```
@KimberleyCook do we want to add spaces around this dash? We have spaces around dashes everywhere else.